derp/derp_server.go

```diff
@@ -1696,31 +1687,18 @@ func (s *Server) ConsistencyCheck() error {
 // It is ~4x slower than io.ReadFull(br, key),
 // but it prevents key from escaping and thus being allocated.
```

Comment:
Have you confirmed that key.ReadFrom prevents key from escaping? (Is this property protected by an allocs-per-run test, and if not, will you add one? Thanks.)